{{cookiecutter.project_slug}}/.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
```

Comment:
none of these should be ignored.
the code inside the template should be fixed instead